packages/shared/components/Profile.svelte

```diff
@@ -53,7 +53,7 @@
                     height={14}
                 />
             {/if}
-            <Text type="h5" classes="text-center">{name}</Text>
+            <Text type="h5" classes="text-center w-28 truncate">{name}</Text>
```

Comment:
Tiny comment here:

I think it would be more flexible if instead of hardcoding here a new width, you inherit from the top father which already has `w-24`. If you pass `w-full` to the 2 node parents of this Text component, it will automatically adapt to the available space from the father, so if we set the top width to `w-40`, it will automatically work

![image](https://user-images.githubusercontent.com/3624944/173591434-2148375e-ee9d-4d93-b9ac-39be48d32c0d.png)